docs/content/getting-started/setup-openfga.mdx

```diff
@@ -53,7 +53,31 @@ All possible configurations and their default values are defined in [config-sche
 You can choose between two storage options:
 
 - In-memory storage. This option is not persistent, i.e. if you restart the server, the data will be lost.
```

Comment:
Can we change this to something like:

OpenFGA supports multiple storage engine options including:

- `memory` - The memory storage engine is the default, but it is not persistent (e.g. data is lost between server restarts).
- `postgres` - The postgres storage engine requires a Postgres instance that the OpenFGA server can reach.

Reply:
Updated. Thank you!